modules/fiftyonedegrees/devicedetection/no_cgo.go

```diff
@@ -0,0 +1,13 @@
+//go:build !cgo
+
+package devicedetection
+
+import (
+	"encoding/json"
+
+	"github.com/prebid/prebid-server/v2/modules/moduledeps"
```

Comment:
should be `"github.com/prebid/prebid-server/v3/modules/moduledeps"`

Reply:
`v3` to merge into master or otherwise `v2` if needs a back port into one of these releases..

Reply:
DONE